tests/test_cp_integration.py

```diff
@@ -100,7 +137,8 @@ def test_dot_genproj_with_loc_mappings(self):
         command = GenProjCommandLine()
         command.main(args)
 
-        self.verify_genproj('DotReaderLib17', 342, app_path)
+        self.verify_genproj(app_path, 342, 559, 2)
```

Comment:
The Swift file changed from 342 to 559 because there was a bug in calculating the LOC when a LOC mappings file was passed. We used to only pick the first item in the loc mappings file to calculate the loc for all modules so it was producing less files.